src/DotNetWorker.Core/Hosting/ServiceCollectionExtensions.cs

```diff
@@ -123,6 +125,13 @@ public static IFunctionsWorkerApplicationBuilder AddFunctionsWorkerCore(this ISe
         /// </summary>
         internal static IServiceCollection AddDefaultInputConvertersToWorkerOptions(this IServiceCollection services)
         {
+            if (_defaultConvertersAdded)
```

Comment:
Should we consider adding a marker to the service collection so there isn't a static state tracking this? In some scenarios, that scope might be too broad.

Reply:
changed the approach here to use an `IConfigureOptions` class which lets us use `TryAddEnumerable`